### ISSUE TITLE
Do you play any games?

### ISSUE THREAD

**Issue by cmc333333** [khandelwal/AskShashank#35]
You've recently spent a great deal of time with your extended family. Did you all play card games or party games or anything of that sort to pass the time? Perhaps you hid and played something on the computer instead?

**Comment by khandelwal** [comment]
I would like to say that we play chess and go while sipping whiskey and discussing Sartre. Or that we're so busy that we don't play games and instead we build artisanal furniture together or make organic ice cream from cows we've milked ourselves.  

But all of those things are not true. 

We play Uno or Jenga, and some card game that I don't know the name of but requires betting with quarters. Sometimes we play Scrabble (although in recent years that's just been my wife and me). 

We own Ticket to Ride, Settlers of Catan, Clue and Dominion but it turns out that you need friends to play those games (for various reasons these don't work with family). I've been looking for friends, but it's hard when people run away from the cheese section at the supermarket when you ask if they'll Puerto Rico with you.

**Comment by khandelwal** [comment]
In retrospect, people might be friendlier in the meat section, but as a vegetarian I have no reason to be there and that might get awkward later in the conversation.

**Comment by mebates** [comment]
I will play Ticket to Ride with you any time @khandelwal. I am very competitive.

**Comment by iuloshi** [comment]
Heads up!

**Comment by jpyuda** [comment]
[Scrabble](http://m.youtube.com/watch?v=pUJJ-mdd9pw)

**Comment by rjmajma** [comment]
I will totally play Ticket to Ride with you. I've never played Settler of Catan though.

**Comment by theresaanna** [comment]
> We own Ticket to Ride, Settlers of Catan, Clue and Dominion but it turns out that you need friends to play those games

You understand me so well, Shashank. I, too, wish to play board games, but the lack of local friends makes it quite difficult. (sshhh. just pretend I have non-local friends. Its like the elusive Canadian girlfriend.)

**Comment by rjmajma** [comment]
Ticket to Ride has a great game app. @michellehood and I have been playing that a bit and, for me, it's more enjoyable than the board game (mostly because I'm lazy and don't like counting train cars and remembering where my stations are, which the game app does for me).

**Comment by michellehood** [comment]
Agreed, the Ticket to Ride app is better for lazy people like myself and @rjmajma. @theresaanna if you download it you can play with friends from anywhere!

**Comment by theresaanna** [comment]
Ugh even more reason to figure out how I can clear space on my phone so I can actually download another app.

That or somehow feel like paying $600 for a new phone makes sense.

This has been a multi-month struggle.

**Comment by stephanieosan** [comment]
I have been living in that space, too, @theresaanna! 

I just removed all the photos off my phone. The results have been glorious.

**Comment by khandelwal** [comment]
When you say you removed ALL the photos off your phone - I'm hoping you mean that you left some of me on there. Because that would be a crime otherwise.

**Comment by michellehood** [comment]
The Sun King will not be deleted! 

![image](https://cloud.githubusercontent.com/assets/6954953/5653443/352e042a-9688-11e4-99e2-042e0c18b2ad.png)

**Comment by stephanieosan** [comment]
Don't worry, they're all still on my computer. 
![shashank](https://cloud.githubusercontent.com/assets/6433178/5653520/e8bb0e3e-9688-11e4-8a23-a28c33ce09d8.JPG)

Sun-pants.
